kafka-connect-s3/src/main/java/io/confluent/connect/s3/S3SinkConnectorConfig.java

```diff
@@ -63,6 +63,9 @@
   public static final String REGION_CONFIG = "s3.region";
   public static final String REGION_DEFAULT = Regions.DEFAULT_REGION.getName();
 
+  public static final String AVRO_CODEC_CONFIG = "avro.codec";
+  public static final String AVRO_CODEC_DEFAULT = "deflate";
```

Comment:
Setting this default would change the existing behavior of the connector (i.e. an existing user would suddenly be getting deflated files instead of uncompressed). I'm also not sure deflate is a default most would want these days (e.g. is the decision based on performance/compression tradeoff? splittability? something else?).